src/transport/SecureSession.h

```diff
@@ -72,11 +72,11 @@ class SecureSession : public Session
 
     // TODO: This constructor should be private.  Tests should allocate a
     // kPending session and then call Activate(), just like non-test code does.
-    SecureSession(Type secureSessionType, uint16_t localSessionId, NodeId peerNodeId, CATValues peerCATs, uint16_t peerSessionId,
-                  FabricIndex fabric, const ReliableMessageProtocolConfig & config) :
+    SecureSession(Type secureSessionType, uint16_t localSessionId, NodeId peerNodeId, NodeId localNodeId, CATValues peerCATs,
```

Comment:
Would it make sense to group localNodeId with localSessionId, so we have the local bits then the peer bits, instead of this interleaving?